gramlib/grammar.ml

```diff
@@ -956,19 +956,19 @@ let name_of_symbol : type s tr a. s ty_entry -> (s, tr, a) ty_symbol -> string =
   | Sself -> "[" ^ entry.ename ^ "]"
   | Snext -> "[" ^ entry.ename ^ "]"
   | Stoken tok -> L.tok_text tok
-  | _ -> "???"
```

Comment:
This is dead code actually, you could directly write an assertion failure.

Reply:
Right. Changed.